src/music.c

```diff
@@ -1259,6 +1295,32 @@ int Mix_PausedMusic(void)
     return (music_active == SDL_FALSE);
 }
 
+void Mix_StartTrack(int track)
+{
+    Mix_LockAudio();
+    if (music_playing && music_playing->interface->StartTrack) {
```

Comment:
This should return an int result code, and have an else clause with:
`result = Mix_SetError("That operation is not supported");`